base/file.jl

```diff
@@ -401,11 +401,12 @@ struct uv_dirent_t
 end
 
 """
-    readdir(dir::AbstractString=".") -> Vector{String}
+    readdir(dir::AbstractString="."; sort::Bool=true) -> Vector{String}
 
 Returns the files and directories in the directory `dir` (or the current working directory if not given).
+If `sort=false` the resulting list won't necessarily be sorted.
```

Comment:
comma before "the"

Reply:
Maybe some comment on why someone would pass `sort=false`: e.g. add (after a comma) "but performance may improve slightly.

Reply:
(On my machine, calling `sort!` slows down `readdir` by about 5%, for either ~40 entries or ~1000 entries.)

Reply:
I had also thought about that. Thanks for the review!